types/types.go

```diff
@@ -153,6 +153,9 @@ type ImageDestination interface {
 	// uploaded to the image destination, true otherwise.
 	AcceptsForeignLayerURLs() bool
 
+	// AcceptsAnyOSSpecificImage returns true if the destination can store images for every operating system. False otherwise.
+	AcceptsAnyOSSpecificImage() bool
```

Comment:
I don’t have _obviously_ better ideas. `AcceptsAnyOSSpecificImage` works fine with me.

`AcceptsImagesForAnyOS`? Still weird, and `Image` is kinda implied by the `ImageDestination`…

Or perhaps naming the unusual thing instead of the common case, `MustMatchRuntimeOS` or something like that?